### PR TITLE
Ch add travis 150397532

### PR DESCRIPTION
**Pull request by THEozmic**

### CODE REVIEW

**Review by mradeybee — CHANGES_REQUESTED** [review]
@THEozmic please make sure that travis run successfully

File: .travis.yml [hunk 1]
```diff
@@ -9,9 +9,8 @@ install:
...
 script:
     - npm test
-services:
-  - mongodb
-cache: 
-  - npm install
 notifications:
   webhooks: $COVERALLS_URL
+before_script:
+  - export DISPLAY=:99.0
```
Comment (mradeybee) [inline .travis.yml:15]: What is the purpose of this?

**Review by mradeybee — CHANGES_REQUESTED** [review]
One question, otherwise it looks good.

File: .travis.yml (hunk 1 above)
Comment (THEozmic) [reply .travis.yml:15]: Karma launches a browser so this is creating a virtual browser

**Review by mradeybee — APPROVED** [review]
GTM!

**Pull request merged**